.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * This is a very stripped-down eslintrc, because @khanacademy/eslint-config
+ * assumes you're using react & flow (and requires a lot of dependencies along
+ * with it).
+ */
+const khan = require("@khanacademy/eslint-config");
+
+// Only keep the rules that have no prefix (so are included in eslint core)
+// and the prettier rule.
```

Comment:
I had a look at `khan/eslint-config-khan` and see that everything's listed as a peer dep.  I wonder if we should change that.